sway-ir/src/printer.rs

```diff
@@ -294,7 +334,7 @@ fn maybe_constant_to_doc(
     maybe_const_val: &Value,
 ) -> Doc {
     // Create a new doc only if value is new and unknown, and is a constant.
-    if !namer.is_known(maybe_const_val) && maybe_const_val.is_constant(context) {
+    if !namer.is_known(maybe_const_val) && (maybe_const_val.is_constant(context)) {
```

Comment:
Extra parens?  Doesn't clippy usually warn about this?

Reply:
Weird indeed. Fixed.